ui/main/src/tests/mocks/CardsServer.mock.ts

```
@@ -12,7 +12,7 @@ import {CardForPublishing} from '@ofServices/cards/model/CardForPublishing';
 import {CardCreationReportData} from '@ofServices/cards/model/CardCreationReportData';
 import {CardsFilter} from '@ofServices/cards/model/CardsFilter';
 import {FieldToTranslate} from '@ofServices/cards/model/FieldToTranslate';
-import {LightCard} from 'app/model/LightCard';
+import {Card} from 'app/model/Card';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Card'. Consider to delete it or rename it.

'Card' is defined here:


(lint/suspicious/noRedeclare)